examples/oidc/james/imapserver.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<imapservers>
+    <imapserver enabled="true">
+        <jmxName>imapserver</jmxName>
+        <bind>0.0.0.0:143</bind>
```

Comment:
Can we just remove the startTLS endpoints and just keep the SSL endpoints? (IMAP & SMTP)